objects/AllPlayerCards.15bb07/BookofLivingMyths.c5fb1f.json

```diff
@@ -58,5 +58,23 @@
     "scaleZ": 1
   },
   "Value": 0,
-  "XmlUI": ""
+  "XmlUI": "\u003cInclude src=\"playercards/BookofLivingMyths.xml\"/\u003e",
+  "CustomUIAssets": [
+    {
+      "Type": 1,
+      "Name": "font_teutonic-arkham",
+      "URL": "http://cloud-3.steamusercontent.com/ugc/2027213118467703445/89328E273B4C5180BF491516CE998DE3C604E162/"
+    },
+    {
+      "Type": 0,
+      "Name": "bless",
+      "URL": "http://cloud-3.steamusercontent.com/ugc/2380784374775547028/7855033DE0EB1FDDF706E1303054D35FE0902532/"
+    },
+    {
+      "Type": 0,
+      "Name": "curse",
+      "URL": "http://cloud-3.steamusercontent.com/ugc/2380784374775547135/2360372CBE9452CB7B4D135BE13BBA6D46B7D427/"
+    }
+    ]
+
```

Comment:
Nitpick: remove empty line and fix indentation for the final bracket (or this will show up as change with the next unbundling)